js/main.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+
+// Функция, возвращающая случайное целое число из переданного диапазона включительно.
+//Использованы материалы: https://learn.javascript.ru/task/random-int-min-max
+
+const getRandomInteger = function (min, max) {
+  if (max <= min) {
+    console.log('Ошибка! Заданы неверные границы диапазона');
+    return undefined;
```

Comment:
```suggestion
    return null;
```
`undefined` лучше никогда не использовать именно в коде. Для обозначения пустоты используй null